xskillscore/core/probabilistic.py

```diff
@@ -573,6 +578,10 @@ def rps(
         one. If False (default), the new object will be returned without attributes.
     member_dim : str, optional
         Name of ensemble member dimension. By default, 'member'.
+    category_dist: str or None
```

Comment:
I don't think this is a clear name for this argument. What about something like `input_distributions` instead?

Reply:
Agree with @dougiesquire. It's longer but may be more clear `input_distribution_function` default to None with options "c" (for cumulative) and "p" (for probabilistic)

Reply:
implemented both